superset/assets/stylesheets/exploreV2/exploreV2.css

```diff
@@ -1,22 +0,0 @@
-.scrollbar-container {
```

Comment:
We should delete the css reference [https://github.com/airbnb/superset/blob/34d0dd9d6e074e41968c39247d66e19fd551c57b/superset/templates/superset/explorev2.html#L12](here)